### PR TITLE
JS pull request -New md files with headers

### LINKED ISSUE #19: Add file for JS
**New file JS**
Please do this ....

### PR DESCRIPTION
**Pull request by qb444**
This pull request resolves #19 (issue no. 19) 
Test of md file

@eq203 

**Still to do:**
- [x] Task 1
- [ ] Task 2

Refers also to link [https://github.com/githubschool/13-dec/issues/1#issuecomment-446898109](url)

### CODE REVIEW

File: hometown/JS-file-1.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## File 1 JS
+This is my file - JS
```
Comment (eq203) [inline hometown/JS-file-1.md]: Great job

File: hometown/JS-file-1.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+## File 1 JS
```
Comment (qb444) [inline hometown/JS-file-1.md]: + Comment on line 1 - JS

File: hometown/JS-file-1.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+This is my file - JS
+- List item
+## Things to do
```
Comment (eq203) [inline hometown/JS-file-1.md]: Hier fehlen Dir noch Einträge

Comment (qb444) [inline hometown/JS-file-1.md]: Coment to line 4
To sports

**Pull request merged**